docs/site/content/docs/tmc-register.md

```diff
@@ -0,0 +1,58 @@
+# Register a Management Cluster in Tanzu Mission Control
+
+VMware Tanzu Mission Control is a licensed VMWare product that you can integrate with.
+
+Use Tanzu Mission Control to manage your entire Kubernetes footprint, regardless of where your clusters reside.
+Bring all of your Kubernetes clusters together to monitor and manage them from a single console backed by an API-driven service.
+
+Complete the following steps to register a Tanzu Community Edition management cluster in Tanzu Mission Control.
```

Comment:
@joshrosso - Can you register mgmt clusters created in all the providers - including Docker?

Reply:
@kcoriordan also worth mention, for vSphere clusters, TMC can only support clusters that use the *Photon* OS image. TMC will not work properly with vSphere clusters using the Ubuntu image.

Reply:
added this now